documentation/ame-guide/modules/ROOT/pages/edit-models.adoc

```diff
@@ -128,6 +131,82 @@ For a new model that only has an Aspect so far:
 
 image::different-constraints.png[Different Constraints]
 
+[[abstract-entity-case]]
+=== Use Abstract Entities
```

Comment:
BAMM supports defining abstract elements (entities and properties) and the concept of inheritance in order to make semantic information reusable. The editor allows to create abstract entities and properties and declare relationships and inheritance via "extends". another element. See https://openmanufacturingplatform.github.io/sds-bamm-aspect-meta-model/bamm-specification/2.0.0/modeling-guidelines.html#declaring-entities for more information about abstract entities and properties.